includes/class-airtable-api.php

```diff
@@ -90,7 +110,7 @@ public function get_classroom_building_slug( string $classroom_building_code ) {
 		$response = $this->get( table: 'Buildings', params: $params, request_resource: $classroom_building_code );
 
 		if ( ! $response || empty( $response ) ) {
-			return null;
+			return array();
```

Comment:
Are you sure you want this to return an empty array? Looks like it returns a string if it's successful.

Reply:
Yes, thank you. I had been trying to unsuccessfully solve a fatal error and forgot to change it back.